src/components/Articles.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react'
+import { getArticles } from '../../utils'
+
+export default function Items() {
+	const [currArticles, setCurrArticles] = useState([])
+
```

Comment:
It would be nice to have a loading state so users know it is working. It isn't essential for the ticket, but definitely would be nice to have and won't take long to implement 🙂